test/functional/legacy/011_autocommands_spec.lua

```diff
@@ -18,10 +18,9 @@ local clear, feed_command, expect, eq, neq, dedent, write_file, feed =
   helpers.clear, helpers.feed_command, helpers.expect, helpers.eq, helpers.neq,
   helpers.dedent, helpers.write_file, helpers.feed
 
-if helpers.pending_win32(pending) then return end
```

Comment:
Removing this caused the `file reading, writing and bufnew and filter autocommands FilterReadPre, FilterReadPost` test to get stuck in Appveyor.